src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -149,6 +158,9 @@ export default Relay.createContainer(Message, {
           name
           email
         }
+        invoice {
```

Comment:
Here is where the `Message` component defines what it needs: just an `invoice` (with the rest of the invoice-specific fields themselves left up to the `Invoice ` one.